src/Firehose.Web/Authors/FrancoisRaminosona.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class FrancoisRaminosona : IAmAMicrosoftMVP
+    {
+        public string FirstName => "Francois";
+        public string LastName => "Raminosona";
+        public string StateOrRegion => "Norway";
+        public string EmailAddress => "framinosona@hotmail.fr";
+        public string ShortBioOrTagLine => "Passionate Xamarin/Microsoft technologies developer";
+        public Uri WebSite => new Uri("https://blog.francois.raminosona.com/");
+        public IEnumerable<Uri> FeedUris
+        {
+            get
+            {
+                yield return new Uri("https://blog.francois.raminosona.com/tag/xamarin/rss/");
```

Comment:
Are all these feeds necessary? We do filter out duplicates, ourselves, but it is just wasteful downloading the same content over and over.

Reply:
Pushed a fix, thanks